aci/resource_aci_rtctrlattrp.go

```diff
@@ -170,13 +249,69 @@ func resourceAciActionRuleProfileUpdate(ctx context.Context, d *schema.ResourceD
 		return diag.FromErr(err)
 	}
 
+	// setRouteTagDn - Operations
+	if setRouteTag, ok := d.GetOk("set_route_tag"); ok {
+		if d.HasChange("set_route_tag") {
```

Comment:
I think the hasChange condition should be around the whole if/else setRouteTag as if it was never configured before why try to delete it?